File: src/sulfurasItem.py
from src.normal_item import *

class Sulfuras(NormalItem):
    
    def __init__(self, name, sell_in, quality):
        Item.__init__(self, name, sell_in, quality)
        
    def updateQuality(self):
        assert self.quality == 80, "quality de %s distinta de 80" % self.__class__.__name__
        pass
        self.setSell_in()

File: gildedrose/src/main/java/edu/craptocraft/gildedrose/domain/SulfurasItem.java
package edu.craptocraft.gildedrose.domain;

public class SulfurasItem extends NormalItem {

    public SulfurasItem(String name, int sell_in, int quality){
        super(name, sell_in, quality);
    }

    @Override
    public void updateQuality() {};
}
